package.json

```diff
@@ -93,22 +94,27 @@
     "zod": "^3.23.8"
   },
   "devDependencies": {
-    "@babel/core": "^7.24.0",
+    "@babel/core": "^7.27.1",
```

Comment:
Did you have to downgrade these for vitest/react testing library?

Reply:
Doh, I can't read. For some reason I read them as a downgrade 🤦🏻. Move along please, nothing to see here!

![nAGT5LZ](https://github.com/user-attachments/assets/3892c87a-5dbe-494f-b27f-081718df1b9d)